pages/solidity-sdk.tsx

```diff
@@ -25,28 +25,28 @@ const GUIDES = [
     link: "https://blog.thirdweb.com/guides/create-a-generative-art-nft-collection-using-solidity-javascript/",
   },
   {
-    title: "Build a Blockchain Game using ContractKit",
+    title: "Build a Blockchain Game using the Solidity SDK",
     image:
       "https://blog.thirdweb.com/content/images/size/w2000/2022/10/Group-625858--1-.png",
     link: "https://blog.thirdweb.com/guides/build-a-blockchain-game-using-contractkit/",
   },
 ];
 
-const TRACKING_CATEGORY = "contract_kit";
+const TRACKING_CATEGORY = "solidity_sdk";
```

Comment:
this will "break" tracking - make sure to update the insights based on this as well

Reply:
Nacho already warned me - this needs fixing for Release -> publish as well btw!